trestle/core/commands/author/catalog.py

```diff
@@ -49,7 +49,9 @@ class CatalogGenerate(AuthorCommonCommand):
     def _init_arguments(self) -> None:
         name_help_str = 'Name of the catalog model in the trestle workspace'
         self.add_argument('-n', '--name', help=name_help_str, required=True, type=str)
-        self.add_argument('-o', '--output', help=const.HELP_MARKDOWN_NAME, required=True, type=str)
+        self.add_argument(
+            '-o', '--output', help='Name of the output generated catalog markdown folder', required=True, type=str
```

Comment:
This is awkward because there is a different constant for each command.  Previously they all referred to the profile markdown folder - which is a bug.  It was a case where using a const actually caused a problem.  So rather than make 5 different consts used once I changed the help text in each.